components/edit-collective/AssignVirtualCardModal.js

```diff
@@ -268,6 +286,31 @@ const AssignVirtualCardModal = ({ collective, host, virtualCard, onSuccess, onCl
                 />
               )}
             </StyledInputField>
+
+            <StyledInputField
+              gridColumn="1/3"
+              labelFontSize="13px"
+              label="What provider do you use for this card?"
```

Comment:
Maybe internationalize this label. 😉

Reply:
This is a separate issue : https://github.com/opencollective/opencollective/issues/4857

Reply:
Ops, missed that one; I forgot my own issue. 😉